src/main/scala/scalismo/ui/api/Views.scala

```diff
@@ -440,6 +440,12 @@ case class LandmarkView private[ui] (override protected[api] val peer: LandmarkN
     peer.opacity.value = o
   }
 
+  def scalingFactor: Double = peer.scaling.value
+
+  def scalingFactor_=(o: Double): Unit = {
```

Comment:
`d` or `s` would be a better short name, but this is just cosmetics and is not a must change for me.